ci-operator/step-registry/optional-operators/ci/run-bundle/optional-operators-ci-run-bundle-commands.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Deploying an operator in the bundle format using operator-sdk run bundle command"
+
+# `run bundle` supports three install modes: AllNamespaces, OwnNamespace and SingleNamespace.
+if [[ "${OO_TARGET_NAMESPACES}" != "AllNamespaces" && "${OO_TARGET_NAMESPACES}" != "OwnNamespace" && "${OO_TARGET_NAMESPACES}" != "SingleNamespace" ]]; then
```

Comment:
FYI `run bundle` will handle this check. It does support `MultiNamespace` so that should still be checked here.